modules/viz/include/opencv2/viz/viz3d.hpp

```diff
@@ -142,6 +142,12 @@ namespace cv
             */
             Camera getCamera() const;
 
+            /** @brief Sets pose the camera roll. This will spin the camera about its axis.
+
+            @param angle The new roll angle in degrees of the camera.
+             */
+            void cameraRoll(double angle);
```

Comment:
I intentionally didn't add such function since this can be done using `setViewerPose();` with more as I think elegant sintax with Affine3d methods.

Suppose you do 180 degree rotation which can be done as `setViewerPose(Affine3d::Identity())`